projects/sandbox/src/app/sandbox-navigation/sandbox-navigation.component.ts

```diff
@@ -608,7 +617,10 @@ export class SandboxNavigatonComponent extends DataPollingComponent implements O
 
         if (this.progressComplete()) {
           this.resetBusy();
-          if (this.progressData[fieldNamePortalPublish]) {
+          if (
+            this.progressData['dataset-logs'].length > 0 ||
```

Comment:
Here you can also check with the status instead

Reply:
Done